src/test/java/com/solvd/BasicHomePageTests.java

```diff
@@ -0,0 +1,32 @@
+package com.solvd;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.solvd.pages.HomePage;
+import com.solvd.pages.SearchPage;
+
+//Implementation of Web Automation Java Task 1
+public class BasicHomePageTests extends AbstractTest {
+    @Test
+    public void checkIfHoveringDisplaysMainMenuTest() {
+        HomePage homePage = new HomePage(getDriver());
+        homePage.navigateToHomePage();
+
+        homePage.hoverOverHomeButton();
+        Assert.assertTrue(homePage.isHomeMenuVisible(), "Failed to display main menu");
```

Comment:
Make assertion messages specific so failures are easier to understand when reading test reports.

"Main menu was not visible after hovering over Home button"